From: Luca Weiss <luca.weiss@fairphone.com>
Date: Fri, 8 Jul 2022 16:05:04 +0200
Subject: ASoC: qcom: sm8250: Increase MI2S BCLK rate for 32-bit playback

Increase the MI2S bclk rate so that 32-bit playback can work on
Fairphone 5 with AW88261 amplifiers.

Signed-off-by: Luca Weiss <luca.weiss@fairphone.com>
---
 sound/soc/qcom/sm8250.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/sound/soc/qcom/sm8250.c b/sound/soc/qcom/sm8250.c
index cb184ef..ca3697cd 100644
--- a/sound/soc/qcom/sm8250.c
+++ b/sound/soc/qcom/sm8250.c
@@ -15,7 +15,8 @@
 #include "usb_offload_utils.h"
 #include "sdw.h"
 
-#define MI2S_BCLK_RATE		1536000
+/* 48000 kHz * 32 bits/sample * 2 channels */
+#define MI2S_BCLK_RATE		3072000
 
 struct sm8250_snd_data {
 	bool stream_prepared[AFE_PORT_MAX];
